doc/intro.md

```diff
@@ -1,19 +1,16 @@
 # Introduction to witan.models.demography
 
 
-`witan.models.demography` is a Clojure library to run demography models.
-
-Ultimately it will be a tool to run:
+`witan.models.demography` is a Clojure library to run demographic models, which may include:
 
 * population projections
-* school roll projections
 * small area projections
 
 
 `witan.models.demography` proposes different ways to run each of those three types of projections.
 The different methodologies depend on your preferences and/or the data available to you.
 
-Note: Both school roll projections and small area projections take as an input population projections.
+Note: The small area projections take as an input population projections.
 
 
 **The content of the library follows this structure:**
```

Comment:
The things listed under fertility, mortality, and migration below don't seem to correspond 100% with either the structure of the library or the MVP diagram or Ben's docs. I think to a new user if would also be confusing what they mean (i.e. is historic fertility rates- births by age of mother data that needs to be input? is it a calculation? is it a step in the model?) Would suggest changing or removing this section.